src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -413,16 +414,16 @@ transactionSign (
             return e;
     }
 
-    if (!tx_json.isMember ("Sequence"))
-        tx_json["Sequence"] = ledgerFacade.getSeq ();
-
     if (!tx_json.isMember ("Flags"))
         tx_json["Flags"] = tfFullyCanonicalSig;
 
     if (verify)
     {
+        if (!tx_json.isMember ("Sequence"))
```

Comment:
Yeah, just note that when writing code that uses composition (classes that use other classes) these static names will become a _cross cutting concern_
http://en.wikipedia.org/wiki/Cross-cutting_concern